validation/telemetry/sample_v4_ping.json

```diff
@@ -10824,7 +10824,15 @@
             "key1": false,
             "key2": true
           }
-        }
+        },
+        "events": [
+          [2147, "ui", "click", "back_button"],
+	  [2213, "ui", "search", "search_bar", "google"],
```

Comment:
I don't know why it's not indented as the line above...I see it's indented in my local editor... 
Do you have an idea why? @Dexterp37